src/Stateless/Reflection/StateInfo.cs

```diff
@@ -27,7 +25,14 @@ public class StateInfo
                 }
             }
 
-            return new StateInfo(stateReperesentation.UnderlyingState, entryActions, exitActions, ignoredTriggers);
+            StateInfo stateInfo = new StateInfo(stateReperesentation.UnderlyingState, ignoredTriggers);
+
+            stateInfo.EntryActions = stateReperesentation.EntryActions.Select(e => e.Description).ToList();
```

Comment:
Can we stick with the initialized-by-constructor pattern here? It does result in quite a large constructor argument list, but makes it less likely we'll forget to pass these through, and thus end up with null-valued properties, somewhere else.